### ISSUE TITLE
WasmExtension queries without height

### ISSUE THREAD

**Issue by tuloski** [cosmos/cosmjs#1416]
With the WasmExtension for QueryClient is not possible to make a query with a specific height as it can be done with ABCIquery.
And I think it's a very useful feature.
I myself very often query smart contracts but get outdated data without knowing.

**Comment by apolloshab** [comment]
agreed that feature would be awesome. posted a similar issue here https://github.com/osmosis-labs/telescope/issues/356

**Comment by dakom** [comment]
> I myself very often query smart contracts but get outdated data without knowing.

I've also run into this problem, though for my use case I'd prefer just knowing the height a query was actually made at, instead of specifying what it should be made at. That way I can do something like "if query height is less than or equal to the last height I got, sleep and try again"

Will open a PR to address this and take it from there... I think it's across a couple different packages
